app/views/overview.html

```diff
@@ -42,18 +42,18 @@
                    ng-if="service.metadata.labels.app || routesByService[service.metadata.name].length || childServicesByParent[service.metadata.name].length">
                 <overview-service-group></overview-service-group>
               </div>
-
-              <div row wrap>
+              
+              <div row wrap class="overview-standalone-service">
```

Comment:
`standalone-service-row`? The current name sounds too much like `standalone-service` used below.